components/infobox/wikis/rainbowsix/infobox_team_custom.lua

```diff
@@ -0,0 +1,96 @@
+---
+-- @Liquipedia
+-- wiki=rainbowsix
+-- page=Module:Infobox/Team/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Team = require('Module:Infobox/Team')
+local Earnings = require('Module:Earnings of')
```

Comment:
we now have a default earnings implementation in the base module
please check if that would be okay for R6 (you might have to set a tiny module for it on R6 that changes some setting values)
if that Earnings implementation fits R6 you can kick the earnings stuff from this infobox team implementation entirely as it will be handled by the base module

Reply:
Implemented default earnings.